editor/components/post-saved-state/index.js

```diff
@@ -69,14 +68,13 @@ export function PostSavedState( { hasActiveMetaboxes, isNew, isPublished, isDirt
 }
 
 export default connect(
-	( state ) => ( {
+	( state, { forceIsDirty, forceIsSaving } ) => ( {
 		post: getCurrentPost( state ),
 		isNew: isEditedPostNew( state ),
 		isPublished: isCurrentPostPublished( state ),
-		isDirty: isEditedPostDirty( state ),
-		isSaving: isSavingPost( state ),
+		isDirty: isEditedPostDirty( state ) || forceIsDirty,
```

Comment:
Minor: In this case the selector implementation is trivial, but caught my attention to have the function call as the first part of the condition, as if we expect `forceIsDirty` to be a simple truthy check, we could put it first and take advantage of [short-circuit evaluation](https://en.wikipedia.org/wiki/Short-circuit_evaluation).